lib/tests/streamlit/cli_test.py

```diff
@@ -38,7 +38,7 @@ class CliTest(unittest.TestCase):
     """Unit tests for the cli."""
 
     def setUp(self):
-        cli.name = "test"
+        cli.name = "streamlit"
```

Comment:
Do you know the reason why this was named `test` before?

Reply:
unsure -- I couldn't find any obvious reason the name had to be changed, so I don't think there was really any need to do so